controller/src/test/java/io/pravega/controller/auth/JWTTester.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.auth;
+
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import java.io.FileNotFoundException;
+import java.io.UnsupportedEncodingException;
+import java.security.cert.CertificateException;
+
+public class JWTTester {
```

Comment:
I don't think we need this. If it's not a unit test, you will likely be the last person to ever run this code after the PR is merged.

Reply:
Removed from git.